### PR TITLE
Increment cvg_count by 1 with each successful run of 'storage config' command with 'cvg' option.

### PR DESCRIPTION
**Pull request by 83bhp**
Two changes
1. Increment cvg_count by 1 with each successful run of 'storage config' command with 'cvg' option.
2. User data type of cvg_count as int for Pillar and as string for confstore

### CODE REVIEW

**Review by sradhanand-pati — APPROVED** [review]

File: lr-cli/cortx_setup/commands/storage/config.py [hunk 1]
```diff
@@ -461,7 +465,13 @@ def run(self, **kwargs):
                     " are missing")
                 raise RuntimeError("Incomplete arguments provided")
 
-            self.update_pillar_and_conf('cvg', self.cvg_count)
+            current_cvg_count = Conf.get (
+                'node_info_index',
+                f'server_node>{self.machine_id}>storage>cvg_count'
+            )
+            if current_cvg_count is MISSED:
```
Comment (mazinamdar) [inline lr-cli/cortx_setup/commands/storage/config.py:472]: I think we used MISSED for pillar data , we can use None

Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storage/config.py:472]: Probably we don't need this check?

**Review by nitin-seagate — APPROVED** [review]
Looks good to me...

File: lr-cli/cortx_setup/commands/storage/config.py [hunk 2]
```diff
@@ -189,6 +189,10 @@ def update_pillar_and_conf(self, key, value):
 
         if(key=='password'):
             value = self.encrypt_password(value)
+
+        if pillar_key_map[key] == "cvg":
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storage/config.py:193]: This is an issue, we need to match key not value....

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
Looks good to me...

**Pull request merged**